integration/create_test.go

```diff
@@ -26,6 +27,43 @@ import (
 	"github.com/FerretDB/FerretDB/integration/setup"
 )
 
+func TestCreateStressPostgres(t *testing.T) {
+	setup.SkipForTigrisWithReason(t, "Tigris needs a schema for collection creation")
+
+	t.Parallel()
+
+	ctx, collection := setup.Setup(t) // no providers there, we will create collections concurrently
+	db := collection.Database()
+
+	collNum := 10
+
+	var wg sync.WaitGroup
+	for i := 0; i < collNum; i++ {
+		wg.Add(1)
+
+		go func(i int) {
+			collName := fmt.Sprintf("stress_%d", i)
+			err := db.CreateCollection(ctx, collName)
+
+			wg.Done()
+			require.NoError(t, err)
+		}(i)
+	}
+
+	wg.Wait()
+
+	// check that all collections were created, and we can query them
+	for i := 0; i < collNum; i++ {
+		t.Run(fmt.Sprintf("check_stress_%d", i), func(t *testing.T) {
+			t.Parallel()
+
+			collName := fmt.Sprintf("stress_%d", i)
+			res := db.Collection(collName).FindOne(ctx, bson.D{})
+			require.Equal(t, mongo.ErrNoDocuments, res.Err())
```

Comment:
For a non-existing collection, we do not get `ErrNoDocuments`, correct? Just want to ensure that `ErrNoDocuments` is an error we get if the collection exists and no document in the collection.

Reply:
Good point! If collection doesn't exist, it gives `ErrNoDocuments` 🙈

Reply:
That actually revealed some other problems, see the recent update :)